exist-parent/pom.xml

```diff
@@ -755,7 +755,7 @@
                 <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
-                    <version>0.8.9</version>
+                    <version>0.8.8</version>
```

Comment:
Why was the version of the jacoco plugin downgraded?

Reply:
Okay I understand that it was due to a separate issue addressed in https://github.com/eXist-db/exist/pull/4855